changelog/5905.bugfix.rst

```diff
@@ -0,0 +1,4 @@
+Remove option ``token_pattern`` from ``CountVectorsFeaturizer``.
+Instead all tokenizers now have the option ``token_pattern``.
+If a regular expression is set, all detected tokens of the tokenizer are further split into tokens according to the
+regular expression.
```

Comment:
This is not always correct now right? Because for tokenizer like spacy we first use spacy's tokenizer and then apply the token pattern, whereas for convert and language model tokenizers we first apply the token pattern and then use the model's tokenizer. I am a bit confused about how to phrase this without making it sound complex.

Reply:
right... let me think about it